packages/esm-appointments-app/src/form/appointments-form.component.tsx

```diff
@@ -176,10 +176,22 @@ const AppointmentsForm: React.FC<AppointmentsFormProps> = ({
       },
       {
         path: ['appointmentDateTime.recurringPatternEndDate'],
-        message: 'A recurring appointment should have an end date',
+        message: t('A recurring appointment should have an end date'),
```

Comment:
Thanks for following up @lucyjemutai  but I think we our best practice is to have a code (something without spaces) here followed by the default text, ie `"t('recurringAppointmentShouldHaveEndDate','A recurring appointment should have an end date').`

Am I correct @denniskigen ?

Reply:
Will we  aslo need to update the translations ? @mogoodrich

Reply:
I don't think so @lucyjemutai ... if you add the above` t('recurringAppointmentShouldHaveEndDate','A recurring appointment should have an end date') `I believe it will automatically add a translation mapping `'recurringAppointmentShouldHaveEndDate': ''A recurring appointment should have an end date"` to the en.json file.